### ISSUE TITLE
EditMOM3 can produce text()-nodes outside of the structure defined by the template

### ISSUE THREAD

**Issue by GVogeler** [icaruseu/mom-ca#509]
Vera reports:

"While moving around a text fragment (`nr. 122 218-219 v. 1205 wird Riccardus erwähnt; nota 4: `) with cut and paste (CTRL+C, CTRL+V) in EditMOM3 I produced at an undefined moment the following XML:

`<cei:title>Le cartulaire</cei:title>, p. 219, nota 4</cei:bibl>.</cei:p>nr. 122 218-219 v. 1205 wird Riccardus erwähnt; nota 4: <cei:quoteOriginaldatierung/>`

i.e. text between `cei:p` and `cei:quoteOriginaldatierung` - an area of the document which is not accessible in EditMOM3. 
Unfortunately the problem is not reproducable."

**Comment by GVogeler** [comment]
Vera reports another case (alothough again without being able to reproduce it):
1. Opening a document with rich markup in EditMOM3
2. Cursor in EditMOM3 set behind the field delimited guillemt (of `cie:issued/cei:placeName`)
3. Ctrl-V to insert text copied from elsewhere in the document
4. Text is inserted after the closing `</cei:placeName>` (where the schema allows to have text, so no validation error is thrown, but the editor normally does not allow allow entering text)
![clipboard01](https://cloud.githubusercontent.com/assets/324006/20091395/d53cdb9c-a592-11e6-8329-72075a3f9ae0.png)

**Comment by GVogeler** [comment]
Possible path to understand the problem: understand which function in EditMOM3 controls whether you can add text after/before the guillemet (`«`)?

**Comment by StephanMa** [comment]
Yesterday i tried several times to reproduce the error without success...

1) Is this on the dev- or live-server?
2) Which browser is in use? Maybe i can test different browsers by myself.

@maburg can you test aswell?

**Comment by GVogeler** [comment]
Even Vera cannot reproduce the incident deliberately :-) The screenshot provided happend to be available only by chance.

> 1) Is this on the dev- or live-server?

live server

**Comment by GVogeler** [comment]
> 2) Which browser is in use?
Firefox 49.0.2

**Comment by GVogeler** [comment]
while working on http://monasterium.net/mom/smg-VSR/SMG_3_AA_III_12/edit

**Comment by GVogeler** [comment]
@maburg reports: 

> im Fall von Vera steht der Text im cei:issued: das ist nur dann möglich, wenn ,wie sie das macht, einen Elementknoten mit Text kopiert, dann wird zwar geschrieben - 'invalid selection', aber ohne Folgen.

**Comment by GVogeler** [comment]
Codemirror fängt Ctrl-V nicht ab? (keymap)

**Comment by yngwi** [comment]
Mir gerade zufällig aufgefallen: Wenn man ein Element kopiert (nicht nur Text), lässt sich dieses per Paste (Maus & Strg+v) außerhalb des schließenden `<<` einfügen. Dies lässt sich dann nur mehr rudimentär korrigieren (autosave) und nach einem neu laden des Editors scheint diese Änderung, da außerhalb des Elementes, gar nicht mehr auf, ist im XML aber noch präsent. K.a. ob das der Grund für die Problematik in diesem Issue ist, aber es scheint mir auf jeden Fall verwandt.

**Comment by GVogeler** [comment]
Ja, genau so hat @maburg den Fehler von Vera reproduzieren können: https://github.com/icaruseu/mom-ca/issues/509#issuecomment-259087892

**Comment by yngwi** [comment]
Ach ja. Danke für den Hinweis.
